### PR TITLE
fix(): Don't expose EventEmitters in Component public API

### PR DESCRIPTION
**Pull request by trik**
cleaned version

refers #440

### CODE REVIEW

File: src/components/radio/radio.ts [hunk 1]
```diff
@@ -20,6 +20,10 @@ import {
 } from '@angular/common';
 import {MdRadioDispatcher} from './radio_dispatcher';
 
+import {
+  Observable
+} from 'rxjs/Rx';
```
Comment (DevVersion) [inline src/components/radio/radio.ts]: Only import `rxjs/Observable`, not the whole library.

File: src/components/checkbox/checkbox.ts [hunk 2]
```diff
@@ -106,7 +107,8 @@ export class MdCheckbox implements ControlValueAccessor {
   @Input() name: string = null;
 
   /** Event emitted when the checkbox's `checked` value changes. */
-  @Output() change: EventEmitter<boolean> = new EventEmitter<boolean>();
+  private _change: EventEmitter<boolean> = new EventEmitter<boolean>();
+  @Output() get change(): Observable<boolean> { return this._change.asObservable(); };
```
Comment (jelbourn) [inline src/components/checkbox/checkbox.ts]: Your formatting is a bit off. It should be
```ts
private _change: EventEmitter<boolean> = new EventEmitter<boolean>();
@Output() get change(): Observable<boolean> { 
  return this._change.asObservable();
}
```

**Comment by jelbourn** [comment]
@trik Apologies, got distracted from this PR. LGTM if you can rebase.

**Comment by trik** [comment]
@jelbourn no problem, rebased

**Comment by trik** [comment]
@jelbourn browserstack testing timed out

**Comment by DevVersion** [comment]
@trik It seems like the rebase failed. You now have `1,061` line additions, which is not correct.

**Pull request closed without merging**